python/paddle/fluid/tests/unittests/benchmark.py

```diff
@@ -40,8 +41,8 @@ def _assert_cpu_gpu_same(self, cpu_outs, gpu_outs, fetch_list, atol):
             expect_t = np.array(item_cpu_out)
             actual = item_gpu_out
             actual_t = np.array(item_gpu_out)
-            var_name = variable if isinstance(variable,
-                                              basestring) else variable.name
+            var_name = variable if isinstance(
+                variable, six.string_types) else variable.name
```

Comment:
what if it's py2 binary type?

Reply:
never meet ```bytes``` here if we decoded the args' name above